src/instances.ts

```diff
@@ -567,6 +567,9 @@ export function isReferencedFile(instance: TSInstance, filePath: string) {
 }
 
 export function getEmitOutput(instance: TSInstance, filePath: string) {
+  if (fileExtensionIs(filePath, typescript.Extension.Dts)) {
```

Comment:
Sorry missed this earlier but for extension use `instance.compiler.Extension.Dts` `typescript` is used only for types for tsloader code.. The compiler = typescript of the compilation

Reply:
done, thanks for your time